packages/rpc-methods/src/permitted/enable.ts

```diff
@@ -24,14 +24,16 @@ export interface EnableWalletResult {
   errors?: SerializedEthereumRpcError[];
 }
 
+/**
+ * `wallet_enable` is a convenience method the requests
```

Comment:
```suggestion
 * `wallet_enable` is a convenience method to call `requestPermissions`, `wallet_installSnaps` and return `eth_accounts`.
```

Reply:
Whoops. Thanks for catching that! d90a902